src/controls/first_person.rs

```diff
@@ -355,27 +364,27 @@ impl FirstPerson {
             }
         }
 
-        self.axes
-            .vertical
-            .map(|a| if let Some(diff) = input.timed(a) {
+        self.axes.vertical.map(|a| {
```

Comment:
@vitvakatu Looks like formatting has regressed again. Will try to fix.

Reply:
Not sure I can fix this, sorry!

```
$ cargo fmt -- --version
0.2.16-nightly ( )
```

Reply:
It's okay, the same version, same changes in my fork.